examples/step-56/step-56.cc

```diff
@@ -554,12 +554,15 @@ namespace Step56
         // sparsity patterns and matrices for each level. The resize()
         // function of MGLevelObject<T> will destroy all existing contained
         // objects.
+        std::set<types::boundary_id> zero_boundary_ids;
         typename FunctionMap<dim>::type boundary_condition_function_map;
```

Comment:
Yes, this one is not used elsewhere, I assumed that the pattern I had noticed with the other two was repeated, and it was still used.